v4/optgen/support/vim/cropt.vim

```diff
@@ -0,0 +1,51 @@
+" Vim syntax file
+" Language:         Andy's optimizer DSL prototype
+" Maintainer:       Radu Berinde <radu@cockroachlabs.com>
+"
+" To use, copy the file to ~/.vim/syntax/
+" Recommended in .vimrc:
+"    autocmd BufNewFile,BufRead *.opt set filetype=cropt tw=0
+
+if exists("b:current_syntax")
+  finish
+endif
+
+syn match parens display '[()\[\]]'
```

Comment:
Did you mean to add the brackets here as well as below in the "brackets" match rule?

Reply:
Oh, these lines were some leftovers, removed